lean4-infoview/src/infoview/index.css

```diff
@@ -129,182 +141,93 @@ select {
     left: -4px;
 }
 
-.popup-menu-text { text-align: center; padding: 5px; vertical-align:top; display:inline-block; }
-
-.popup-menu-icon { padding: 0px; margin-top:6px; }
-
-.codicon-blank {
-    width: 20px;
-    height: 16px;
+/* Contents of a list of selections shown as a tooltip. */
+.tooltip-menu-content {
+    color: var(--vscode-editorHoverWidget-foreground);
+    background: var(--vscode-editorHoverWidget-background);
 }
 
-.popup-menu {
-    color: var(--vscode-editorWidget-foreground);
-    background: var(--vscode-editorWidget-background);
+.tooltip-menu-content .tooltip-menu-text {
+    text-align: center;
+    padding: 5px;
+    vertical-align: top;
+    display: inline-block;
 }
 
-.hover-div {
-    overflow: hidden;
-    line-height: 1.35714;
+.tooltip-menu-content .tooltip-menu-icon {
+    padding: 0px;
+    margin-top:6px;
 }
 
 /*---------------------------------------------------------------------------------------------
  *  Copyright (c) Microsoft Corporation. All rights reserved.
  *  Licensed under the MIT License. See License.txt in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
- /* from github.com/microsoft/vscode/src/vs/base/browser/ui/hover/hover.css */
- .monaco-hover {
-	cursor: default;
-	overflow: hidden;
-	z-index: 50;
-	user-select: text;
-	-webkit-user-select: text;
-	-ms-user-select: text;
-	box-sizing: initial;
-	line-height: 1.5em;
-}
-
-.monaco-hover.hidden {
-	display: none;
-}
-
-.monaco-hover a:hover {
-	cursor: pointer;
-}
+/* adapted from https://github.com/microsoft/vscode/blob/main/src/vs/base/browser/ui/hover/hover.css */
 
-.monaco-hover .hover-contents:not(.html-hover-contents) {
-	padding: 4px 8px;
+/* Contents of a code tooltip showing a value, type, documentation, etc. */
+.tooltip-code-content {
+   color: var(--vscode-editorHoverWidget-foreground);
+   background: var(--vscode-editorHoverWidget-background);
+   line-height: var(--vscode-editor-line-height);
+   overflow: hidden;
 }
 
-.monaco-hover .markdown-hover > .hover-contents:not(.code-hover-contents) {
-	/* max-width: 500px; */
-	word-wrap: break-word;
+.tooltip-code-content a:hover {
+    cursor: pointer;
 }
 
-.monaco-hover .markdown-hover > .hover-contents:not(.code-hover-contents) hr {
-	/* min-width: 100%; */
+.tooltip-code-content p,
+.tooltip-code-content .code,
+.tooltip-code-content ul {
+    margin: 8px 0;
 }
 
-.monaco-hover p,
-.monaco-hover .code,
-.monaco-hover ul {
-	margin: 8px 0;
-}
-
-.monaco-hover code {
-	font-family: var(--monaco-monospace-font);
```

Comment:
Note: as the webview is its own context, `--monaco-` CSS variables are not defined there and we can't use them.

Reply:
got it, thanks.